src/Launcher.cpp

```diff
@@ -63,37 +64,51 @@ int Launcher::startGUI() {
 struct CoutProgressIndicator : public ProgressIndicator {
     virtual void advance(int percent, const char * message, const char * arg) {
         if (arg) {
-            Log::progress('[', setw(3), percent, "%] ", QCoreApplication::translate("LoadSave", message).arg(arg));
+            Log::progress('[', std::setw(3), percent, "%] ", QCoreApplication::translate("LoadSave", message).arg(arg));
         } else {
-            Log::progress('[', setw(3), percent, "%] ", QCoreApplication::translate("LoadSave", message));
+            Log::progress('[', std::setw(3), percent, "%] ", QCoreApplication::translate("LoadSave", message));
         }
     }
 };
 
 
-list<LoadOptions> Launcher::getBracketedSets() {
-    list<LoadOptions> result;
-    list<pair<ImageIO::QDateInterval, QString>> dateNames;
-    for (QString & name : generalOptions.fileNames) {
-        ImageIO::QDateInterval interval = ImageIO::getImageCreationInterval(name);
-        if (interval.start.isValid()) {
-            dateNames.emplace_back(interval, name);
-        } else {
-            // We cannot get time information, process it alone
-            result.push_back(generalOptions);
-            result.back().fileNames.clear();
-            result.back().fileNames.push_back(name);
+std::list<LoadOptions> Launcher::getBracketedSets() const {
+    std::list<LoadOptions> result;
+    if (generalOptions.grouping == LoadOptions::Grouping::MANUAL) {
+        QStringList fileNames = generalOptions.fileNames;
+        while(!fileNames.empty()) {
+            LoadOptions opt = generalOptions;
+            auto oIt = opt.fileNames.begin();
+            auto goIt = fileNames.begin();
```

Comment:
'opt' is clear here so you could keep it this way but `oIt` and `goIt` are ambiguous specially that they have `auto` data type

Reply:
F**** the last commit destroyed the logic... Thx for forcing me to look at this again.
I wanted this to be compact but readability obviously suffered a bit (or I'm toot tired...) with all the different LoadOptions and fileNames flying around...
I have a fix coming (+ added some comments in the source code).

Reply:
fixed the bug and gave the iterator a more descriptive name.
Is it easier to get now what the code is doing?

Also made sure that both branches -- gt and gm -- are working.

Reply:
awesome! it's much better specially for c++ noobs like me.
I'd say this pr is ready to be merged

Reply:
Great, thanks! :+1: 
Before merging I will have to squash this mess in 3 commits.
I hope I get it done this afternoon. Have to read some HowTos as I'm a total git-noob :worried: